airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -506,9 +488,12 @@ def __init__(  # pylint: disable=too-many-arguments
                 del kwargs['params']
 
             # Create cluster object from kwargs
-            kwargs['region'] = region
```

Comment:
Is `region` unnecessary for `ClusterGenerator` anymore?

Reply:
Yes, I will remove it from ClusterGenerator constructor